windows/src/global/delphi/general/PackageInfo.pas

```diff
@@ -122,19 +122,16 @@   TPackageRegistryKey = class
     property ValueType: TPackageRegistryKeyValueType read FValueType write FValueType;
   end;
 
-  TPackageRegistryKeyList = class(TObjectList)
+  TPackageRegistryKeyList = class(TObjectList<TPackageRegistryKey>)
   private
     FPackage: TPackage;
-    function Get(Index: Integer): TPackageRegistryKey;
-    procedure Put(Index: Integer; const Value: TPackageRegistryKey);
   public
     constructor Create(APackage: TPackage);
     procedure Assign(Source: TPackageRegistryKeyList); virtual;
     procedure LoadIni(AIni: TIniFile); virtual;
     procedure SaveIni(AIni: TIniFile); virtual;
     procedure LoadXML(ARoot: IXMLNode); virtual;
     procedure SaveXML(ARoot: IXMLNode); virtual;
-    property Items[Index: Integer]: TPackageRegistryKey read Get write Put; default;
```

Comment:
Are these implemented automatically by the generic base class?  I assume that's why they're being removed.

Reply:
Correct